README.md

```diff
@@ -41,4 +41,4 @@ try installing another.
 
 ## Contribution
 
-See [Contributing Guide](https://github.com/discordjs/voice/blob/master/.github/CONTRIBUTING.md).
+See [Contributing Guide](https://github.com/discordjs/voice/blob/main/.github/CONTRIBUTING.md).
```

Comment:
```suggestion
See [Contributing Guide](./.github/CONTRIBUTING.md).
```

Reply:
doesn't look like that works on https://discordjs.github.io/voice/